src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_ARM/ARMReadyToRunGenericHelperNode.cs

```diff
@@ -245,14 +233,14 @@ protected sealed override void EmitCode(NodeFactory factory, ref ARMEmitter enco
                 case ReadyToRunHelperId.VirtualDispatchCell:
                 case ReadyToRunHelperId.DefaultConstructor:
                     {
-                        ARMDebug.EmitNYIAssert(factory, ref encoder, "general EmitCode is not implemented");
+                        // ARMDebug.EmitNYIAssert(factory, ref encoder, "general EmitCode is not implemented");
```

Comment:
Is it intentional that this is enabled, but the NOT TESTED comment is still here?